sig-usability/charter.md

```diff
@@ -0,0 +1,52 @@
+# SIG Usability Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+The scope of SIG usability is the core end-user usability of the Kubernetes project. Thiis covers
+topics like user experience and accessibility. The goals of the SIG are to ensure that the Kubernetes
+project is maximaly usable by users of a variety of different backgrounds and abilities. Examples
+of efforts include internationalization and accessibility of documentation.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- usability of end-user facing experiences (error messages, end-to-end tasks, etc)
+- accessibility of Kubernetes community artifacts, examples include:
+   - internationalization of documentation in binaries and documentation
+   - color choices for people with color blind-ness
+   - ensuring compatability with screen reader technology
+- user interface design for core components with user interfaces.
+
+#### Cross-cutting and Externally Facing Processes
+
+- None currently, eventually perhaps a more formal usability review for changes
+
+### Out of scope
+
+* Contributor user experience (covered by SIG-Contributor Experience)
+* API Design (covered by SIG-Architecture)
+* Command line tool infrastructure (covered by SIG-CLI)
```

Comment:
I would expect that this SIG make efforts towards improving the command line tools' input and output interfaces.

For example, the addition of `-A` has saved me countless keystrokes since its introduction.

Reply:
Perhaps "advisory capacity" for those topics? I agree those sigs should own those components, but it would be good to codify the subject-authority of the SIG.

Reply:
SIGs only have authority over their own sub-projects, and are limited to influence on related topics, so "advisory capacity" makes sense. This SIG should be actively identifying areas of the project where they can influence decision-making to include/increase accessibility. This is similar to contribx where they ensure the contributor is considered in process and other changes.

Reply:
Also not all CLI tools are even SIG-CLI necessarily, e.g. minikube.

However, I'd love to see _someone_ own common tooling (as opposed to kubectl internal) for our CLIs for these problems. More concretely I'd like to see tooling we can use for [kind](https://github.com/kubernetes-sigs/kind) at some point...
I would hope this would be in-scope for a SIG.

Reply:
I added some of this advisory stuff in the section above.

@BenTheElder I think that general CLI tooling libraries, etc. are out of scope for this SIG, though clearly useful for someone to produce :)

Reply:
@BenTheElder Depending on what you are envisioning, https://github.com/kubernetes/cli-runtime/ could be a place to put shared libraries for developing clis.